src/components/EmailEditor/store/emailEditorConfig.js

```diff
@@ -133,6 +172,14 @@ const getDefaultState = () => {
 			},
 			blockLibraries: [],
 			addons: {
+				components: {
+					id: "Components System",
+					description:
```

Comment:
is this needed?

Reply:
you can delete it if you want to, I think we just left it there because we forgot about it